vue/README.md

```diff
@@ -18,15 +18,10 @@ Then, use it as follows:
 // main.js
 
 import Vue from 'vue'
-import App from './app.vue'
 import textMask from 'vue-text-mask'
 
 Vue.directive('text-mask', textMask)
 
-new Vue({
-  el: 'body',
-  components: { App }
-})
```

Comment:
It is needed to bootstrap the component. I didn't see it as being relevant to the text mask directive though. I have changed it back